linkerd/dns/Cargo.toml

```diff
@@ -10,8 +10,14 @@ futures = "0.3"
 linkerd2-dns-name = { path = "./name" }
 linkerd2-error = { path = "../error" }
 tracing = "0.1.19"
-tokio = { version = "0.2", features = ["rt-core", "sync"] }
+tokio = { version = "0.3", features = ["rt", "sync"] }
 pin-project = "0.4"
+# trust-dns depends on Tokio 0.2
+# TODO(eliza): when `trust-dns-resolver` supports Tokio 0.3, remove this.
```

Comment:
It looks like this has been fixed on master https://github.com/bluejekyll/trust-dns/blob/main/crates/resolver/Cargo.toml#L80